bundle/manifests/quay-operator.clusterserviceversion.yaml

```diff
@@ -2,13 +2,13 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    olm.skipRange: ">=3.4.x <3.5.7"
+    olm.skipRange: ">=3.4.x <3.5.8"
     capabilities: Full Lifecycle
     categories: Integration & Delivery
-    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-operator-rhel8:v3.5.7
-    createdAt: 2021-09-09 20:09 UTC
+    containerImage: registry-proxy.engineering.redhat.com/rh-osbs/quay-quay-operator-rhel8:v3.5.8
```

Comment:
I would say it would be better to make these image references point to the image pushed by the github action, wdyt? i.e. whatever is in this CSV must deploy the current 3.5 tag

Reply:
definitely!